app/main.py

```diff
@@ -1,13 +1,28 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {"line": error["line_number"], "column": error["column_number"],
+            "message": error["text"], "name": error["code"],
+            "source": "flake8"} if len(error) > 0 else {}
```

Comment:
```suggestion
    return {
        "line": error["line_number"],
        "column": error["column_number"],
        "message": error["text"],
        "name": error["code"],
        "source": "flake8"
    }
```

Please, place one item per line for this and following cases

Reply:
No reason for length check in this task